index.pug

```diff
@@ -0,0 +1,2366 @@
+                )
+              .testi-msg
+                p.msg
+                  | Creating No Cost EMI offers through the Razorpay dashboard was surprisingly easy. We saw ~3x increase of the EMI contribution in the overall sales.
+                ul
+                  li
+                    h5 Shubham Kumar
+                    p Product Manager
+                  li.testi-logo
+                    img(
+                      src="https://demo.dvio.global/lp/razorpay/pug/images/unacademy-logo.jpg"
+                    )
+                  li.quotes
+                    img(
+                      src="https://demo.dvio.global/lp/razorpay/pug/images/quotes.png"
+                    )
+        .one-slide
+          .testimonial.testi1
+            .message
+              .testi-img
+                img(
+                  src="https://demo.dvio.global/lp/razorpay/pug/images/testi-two.jpg"
+                )
+              .testi-msg
+                p.msg
+                  | Through Payment Pages and Payment Buttons we could start accepting payments just by adding a link to a page or script on the webpage.
+                ul
+                  li
+                    h5 Anubha Maneshwar
+                    p Founder, Girlscript
+                  li.testi-logo
+                    img(
+                      src="https://demo.dvio.global/lp/razorpay/pug/images/girl-script-logo.jpg"
+                    )
+                  li.quotes
+                    img(
+                      src="https://demo.dvio.global/lp/razorpay/pug/images/quotes.png"
+                    )
+        .one-slide
+          .testimonial.testi1.video
+            .message
+              img.VidTesti(
+                src="https://demo.dvio.global/lp/razorpay/pug/images/video-testi.png"
+              )
+              .testi-msg
+                ul
+                  li
+                    h5 Anubha Maneshwar
+                    p Founder, Girlscript
+                  li.testi-logo
+                    img(
+                      src="https://demo.dvio.global/lp/razorpay/pug/images/logos/1.png"
+                    )
+                  li.quotes
+                    img(
+                      src="https://demo.dvio.global/lp/razorpay/pug/images/quotes-w.png"
+                    )
+      .controller
+        .control.prev(style="top: 0px; right: 0px")
+          img(src="https://demo.dvio.global/lp/razorpay/pug/images/arrow.svg")
+        .control.next(style="top: 0px; left: 20px")
+          img(src="https://demo.dvio.global/lp/razorpay/pug/images/arrow.svg")
+  .custom-shape-divider-bottom-1632475645
+    svg(
+      data-name="Layer 1",
+      xmlns="http://www.w3.org/2000/svg",
+      viewbox="0 0 1200 120",
+      preserveaspectratio="none"
+    )
+      path.shape-fill(d="M1200 120L0 16.48 0 0 1200 0 1200 120z")
+// Testimonial Section End
+// Faq Section start
+section#faqs
```

Comment:
replace the FAQ'S Section where below code and update the content according to yours
```suggestion
section#faqSection
    .container
      .faq-container
        .faq
          -
            var faqList = [
              {
                q: "What is the difference between UPI QR Code and Bharat QR code?",
                a: "UPI QR Code accepts all UPI based payments. UPI QR can be used to make (person to person) as well as (person to business) transaction. Bharat QR accepts both UPI as well as credit and debit card based payments. Bharat QR supports only (person to business) transactions. Therefore, only business can get a BharatQR code."
              },
              {
                q: "What is a static and dynamic QR code?",
                a: " A static QR code is an open QR code that can accept any amount of payment. The customer will have to enter the amount at the time of payment. A dynamic QR code has a fixed amount linked to the QR code. A customer will be able to pay only the fixed amount assigned to the QR code. "
              },
              {
                q: "What is the benefit of using a QR code?",
                a: "Cheaper and easy to adopt -  Compared to a POS machine (cost upto Rs. 12,000)<br/><strong>Easy to manage and maintain</strong> - Generate, Print and Display<br/><strong>Quick and contactless payments</strong> - No need to handle cash or change <br/><strong>Better success rates</strong> - Scan and Pay! Less manual entries as compared to cards <br/><strong>Compliance with Govt regulations</strong> - Adopt Dynamic QR codes as per GST norms and <a href='https://cbic-gst.gov.in/pdf/Circular_Refund_146.pdf'>click here</a> to refer the circular. To integrate with our APIs refer <a href='https://razorpay.com/docs/qr-codes/'>product documentation</a>."
              },
              {
                q: "What is the advantage of Razorpay QR codes over others?" ,
                a: "Branded QR code  - Business logo, name and UPI handle to improve brand visibility.<br/><strong>Customized QR code</strong> - Choose from 7+ features, get QR code for your business needs<br/><strong>Unlimited QR codes free of cost</strong> - Get multiple QR codes for different locations, customers, sales executives etc. <br/><strong>Track and reconcile payments</strong> - Use dashboard, reports and webhooks, save valuable time. <br/><strong>All your payments under one roof</strong> - Collect payments from all modes with Razorpay. "
              },
              {
                q: "Which government bodies regulate the usage of UPI QR and Bharat QR?" ,
                a: "  UPI QR was developed by National Payments Corporation of India (NPCI) and regulated by Reserve Bank of India (RBI).<br/><br/>Bharat QR has been developed by NPCI, Mastercard, Visa. It is regulated under instructions by RBI."
              },
              {
                q: "Can Bharat QR be customized with a business logo and design?" ,
                a: "No, since Bharat QR supports both UPI and Cards, there is restriction on branding and display. Therefore, only business name and address can be displayed with restrictions on branding."
              },
              {
                q: "How can I get Bharat QR code?" ,
                a: "You will have to raise a request with the Razorpay support team to activate Bharat QR on your dashboard. The default offering is a UPI QR code."
              },
            ]

          include /partials/faqs
          +faqs(faqList)  
```